### PR TITLE
[TH] Remove interpunction from Thai articles

### PR DESCRIPTION
**Pull request by C0NS03L**
since I just knew that Thai doesn't use commas.
and I removed some full stop I missed

### CODE REVIEW

**Comment by Trigonoculus** [comment]
There's also some outstanding issues in the articles that we can probably fix. Not sure how I can add commits, if not I will work on this after this PR is merged.

File: wiki/Accuracy/th.md [hunk 1]
```diff
@@ -23,49 +23,49 @@
...
+ใน osu!taiko ความแม่นยำจะถูกคำนวนโดยการบวกความแม่นยำของแต่ละโน้ต (ผู้เล่นกดตรงโน้ตแค่ไหน) หารด้วยจำนวนของโน้ตทั้งหมด ถ้าความแม่นนำของโน้ตขึ้นว่า GREAT (良) (นับเป็น 100%) GOOD (可) (นับเป็น 50%) (half) and MISS/BAD (不可) (นับเป็น 0% และจะรีเซ็ทคอมโบ) Drum rolls และ Spinners ไม่มีผลต่อความแม่นยำ
 
 ### ![](/wiki/shared/mode/catch.png) osu!catch
 
 ![ความแม่นยำ = (จำนวนของ droplets + จำนวนของ drops + จำนวนของ fruits) / (จำนวนของ missed droplets + จำนวนของ missed drops + จำนวนของ missed fruits + จำนวนของ droplets + จำนวนของ drops + จำนวนของ fruits)](img/accuracy_catch.png "สูตรการคำนวนความแม่นยำของ osu!catch")
 
-ใน osu!catch, ความแม่นยำจะถูกคำนวนโดย ใช้จำนวนของ Hit objects ที่เก็บได้ทั้งหมดที่ไม่ใช่ Spinner , หารด้วยจำนวนของ hit objects ทั้งหมด. Hit objects ทุกอันมีค่าเท่ากัน; ยกเว้นกล้วย เพราะว่ามันเป็นส่วนหนึ่งของ Spinner
+ใน osu!catch ความแม่นยำจะถูกคำนวนโดย ใช้จำนวนของ Hit objects ที่เก็บได้ทั้งหมดที่ไม่ใช่ Spinner  หารด้วยจำนวนของ hit objects ทั้งหมด Hit objects ทุกอันมีค่าเท่ากัน; ยกเว้นกล้วย เพราะว่ามันเป็นส่วนหนึ่งของ Spinner
```
Comment (Trigonoculus) [inline wiki/Accuracy/th.md:32]:
```suggestion
ใน osu!catch ความแม่นยำจะถูกคำนวนโดย ใช้จำนวนของ Hit objects ที่เก็บได้ทั้งหมดที่ไม่ใช่ Spinner หารด้วยจำนวนของ hit objects ทั้งหมด Hit objects ทุกอันมีค่าเท่ากัน; ยกเว้นกล้วย เพราะว่ามันเป็นส่วนหนึ่งของ Spinner
```
Whitespace

File: wiki/Accuracy/th.md [hunk 2]
```diff
@@ -23,49 +23,49 @@
...
-*โน้ต: ข้อมูลเพิ่มเติมสามารถดูได้เฉพาะ หลังจากเล่น Beatmap นั้น หรือ ดู replay หลังจากออก [หน้าแสดงผล](/wiki/results_screen), ข้อมูลส่วนนี้จะไม่ถูกเซฟ.*
+*โน้ต: ข้อมูลเพิ่มเติมสามารถดูได้เฉพาะ หลังจากเล่น Beatmap นั้น หรือ ดู replay หลังจากออก [หน้าแสดงผล](/wiki/results_screen) ข้อมูลส่วนนี้จะไม่ถูกเซฟ*
 
 ### ความแม่นยำ
 
-เมื่อนำ cursor ไปชี้บนกราฟประสิทธิภาพ, จะแสดง เรตติ้งของ `ความผิดพลาด` and `อัตราความไม่มั่นคง`.
+เมื่อนำ cursor ไปชี้บนกราฟประสิทธิภาพ จะแสดง เรตติ้งของ `ความผิดพลาด` and `อัตราความไม่มั่นคง`
 
-เพราะว่า ม้อด [DT](/wiki/DT) (Double Time) และ [HT](/wiki/HT) (Half Time) มีการนำมาใช้ ,ค่าของ ความผิดพลาด และ อัตราความไม่มั่นคง จะถูกคูณด้วยตัวคูณเดียวกันกับของแมป เพื่อที่จะได้ค่าแท้จริงของ DT, และหารผลลัพธ์ด้วย 1.5 ซึ่งคล้ายกับการคูณผลลัพธ์ด้วย 1.33 เมื่อเล่น HT
+เพราะว่า ม้อด [DT](/wiki/DT) (Double Time) และ [HT](/wiki/HT) (Half Time) มีการนำมาใช้ ,ค่าของ ความผิดพลาด และ อัตราความไม่มั่นคง จะถูกคูณด้วยตัวคูณเดียวกันกับของแมป เพื่อที่จะได้ค่าแท้จริงของ DT และหารผลลัพธ์ด้วย 1.5 ซึ่งคล้ายกับการคูณผลลัพธ์ด้วย 1.33 เมื่อเล่น HT
```
Comment (Trigonoculus) [inline wiki/Accuracy/th.md:54]:
```suggestion
เพราะว่า ม้อด [DT](/wiki/DT) (Double Time) และ [HT](/wiki/HT) (Half Time) มีการนำมาใช้ค่าของ ความผิดพลาด และ อัตราความไม่มั่นคง จะถูกคูณด้วยตัวคูณเดียวกันกับของแมป เพื่อที่จะได้ค่าแท้จริงของ DT และหารผลลัพธ์ด้วย 1.5 ซึ่งคล้ายกับการคูณผลลัพธ์ด้วย 1.33 เมื่อเล่น HT
```
I don't think this makes any sense (doesn't match the English original). Let's fix the stray comma for now.

**Review by Trigonoculus — CHANGES_REQUESTED** [review]

File: wiki/Accuracy/th.md (hunk 2 above)
Comment (C0NS03L) [reply wiki/Accuracy/th.md]: Will fix it in another PR

**Review by Trigonoculus — APPROVED** [review]
Changes look good. We can clean-up these articles after this PR is merged.

**Review by TicClick — APPROVED** [review]

**Pull request merged**